.travis.yml

```diff
@@ -4,12 +4,19 @@ branches:
     - /^v[0-9.]+$/
     - /^maci-.*$/
 language: python
-cache: pip
+cache:
+  pip: true
+  directories:
+    - .tox/
+    - $HOME/.cache/pip-faster/
+    - $HOME/.pre-commit/
```

Comment:
`cache: pip: true` [does that.](https://docs.travis-ci.com/user/caching/#pip-cache)